### PR TITLE
Feature - move pil licence number to profile

### PR DESCRIPTION
**Pull request by joefitter**
* update indexer and search to use pilLicenceNumber from profile

### CODE REVIEW

**Comment by joefitter** [comment]
WIP until the https://github.com/UKHomeOffice/asl-schema/pull/358 migration has run, and related PRs merged

**Review by lennym — APPROVED** [review]

**Review by lennym — CHANGES_REQUESTED** [review]
I can't add a comment or a suggestion because it's not in the diff, but the arguments for `withGraphFetched` on L116 of lib/indexers/profiles/index.js should have `pil` removed.

**Comment by joefitter** [comment]
ah ok, thanks

File: lib/indexers/profiles/index.js [hunk 1]
```diff
@@ -87,14 +84,6 @@ const reset = esClient => {
...
                   }
                 }
-              },
-              pil: {
-                properties: {
-                  licenceNumber: {
-                    type: 'keyword',
-                    normalizer: 'licenceNumber'
-                  }
-                }
```
Comment (wheelsandcogs) [inline lib/indexers/profiles/index.js:97]: Does this not need to become a mapping on `pilLicenceNumber` ?

**Pull request merged**